openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -398,7 +398,7 @@ def login_user(request):
         return JsonResponse(error.get_response())
 
 
-@ensure_csrf_cookie
+@csrf_exempt
```

Comment:
Reminder to add a comment to explain the exemption.

Reply:
@douglashall: How do you feel about this addition to the comment?
```
# CSRF protection is not needed here because the only side effect
# of this endpoint is to refresh the cookie-based JWT, and attempting
# to get a CSRF token before we need to refresh adds too much
# complexity.
```